nlmod/gwt/prepare.py

```diff
@@ -0,0 +1,60 @@
+def set_default_transport_parameters(ds, transport_type):
```

Comment:
Can also add this function to gwt.py and not make a separate module?

Reply:
I like it being somewhat separate even if it is a very small module.